OWML.SampleMods/OWML.LoadCustomAssets/LoadCustomAssets.cs

```diff
@@ -48,6 +54,9 @@ public override void Configure(IModConfig config)
             var speed = config.GetSettingsValue<float>("speed");
             var power = config.GetSettingsValue<float>("power");
             var enableSuperMode = config.GetSettingsValue<bool>("enableSuperMode");
+            var selectedIndex = config.GetSettingsValue<ABC>("thing");
```

Comment:
forgot to rename to `selectedEnum`?

Also, include an example of a setting with numeric options.